config/config.go

```diff
@@ -34,6 +34,7 @@ const (
 	defaultAPIPort             = uint16(8080)
 	defaultMetricsPort         = uint16(9090)
 	defaultIntervalSeconds     = uint64(10)
+	defaultSignatureCacheSize  = uint(1024 * 1024)
```

Comment:
I think we may want a cache size much smaller than this. 1M entries could be gigabytes worth of data that we'd have in memory, and the problem we're solving here is mostly an optimization for validator churn, so most entries should be needed for a relatively short amount of time.